packages/cloud_firestore/cloud_firestore/lib/src/load_bundle_task.dart

```diff
@@ -0,0 +1,13 @@
+part of cloud_firestore;
+
+class LoadBundleTask {
+  LoadBundleTask._(this._delegate) {
+    LoadBundleTaskPlatform.verifyExtends(_delegate);
+  }
+
+  final LoadBundleTaskPlatform _delegate;
+
+  Stream<LoadBundleTaskSnapshot> get stream {
+    return _delegate.stream.map((event) => LoadBundleTaskSnapshot._(event));
```

Comment:
That implies `task.stream != task.stream`, which is a bad practice.

Is it possible to do `late final stream = _deletate.stream.map`?

Or instead do `task.stream()`?